dashboard/app/views/layouts/_school_info_confirmation_dialog.html.haml

```diff
@@ -0,0 +1,25 @@
+-# Rendering this partial causes the school info confirmation dialog to be shown to the
```

Comment:
This is very similar to _school_info_interstitial.html.haml. I wonder if it's possible to share some of this code?

Reply:
> When you merge this would school confirmation dialogues effectively be launched?

***  It will be hidden behind an experiment flag until we are ready to launch.  There is another PR for the addition of the experiment flag.

Reply:
I decided to create another haml file for SchoolInfoConfirmationDialog because there are two different components that are mounted and both components are not rendered at the same time.

Reply:
> I decided to create another haml file for SchoolInfoConfirmationDialog because there are two different components that are mounted and both components are not rendered at the same time.

Separate files makes sense but it would still be nice if some of this code was deduplicated (in a follow up PR). Brad's comment below makes a lot of sense for this!